_header

```diff
@@ -0,0 +1,10 @@
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+Pragma: no-cache
+Expires: 0
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-XSS-Protection: 1; mode=block
+referrer-policy: no-referrer, strict-origin-when-cross-origin
+Strict-Transport-Security: max-age=31536000 ; includeSubDomains
```

Comment:
I don't want to include the subDomains just yet. We don't know if we want to use any of the subdomains for some special purpose which doesn't support HTTPS.

Reply:
I can't think of any purpose on our modern web not supporting HTTPS -especially as Chrome will mark non-HTTPS sites as not secure and certificates are free.

Reply:
You convinced me. Open-source projects should always be leading examples.